source/uwp/Renderer/lib/AdaptiveCard.cpp

```diff
@@ -194,14 +195,15 @@ namespace AdaptiveNamespace
         return S_OK;
     }
 
-    HRESULT AdaptiveCard::get_BackgroundImage(_Outptr_ HSTRING* backgroundImage)
+    _Use_decl_annotations_ HRESULT AdaptiveCard::get_BackgroundImage(IAdaptiveBackgroundImage** backgroundImage)
```

Comment:
>_Use_decl_annotations_ [](start = 4, length = 22)

Don't use _Use_decl_annotations_, add the annotations to both the declaration and the definition